package.json

```diff
@@ -41,6 +41,10 @@
   "dependencies": {
     "classnames": "^2.2.6",
     "jest-environment-node": "^26.0.1",
-    "photoswipe": "^4.1.3"
+    "photoswipe": "^4.1.3",
+    "font-picker-react": "^3.5.2",
```

Comment:
These added deps are only used in the theme editor script, which is now loaded separately from `frontendIndex.js` only if the feature is on.